cloud-aws/src/main/resources/templates/aws-cf-stack.ftl

```diff
@@ -533,6 +536,28 @@
         </#if>
         "ThroughputMode": "${efsFileSystem.throughputMode}"
       }
+    },
+    "MountTarget": {
```

Comment:
should we put here enableEfs as well ?

Reply:
Yes. It should be inside "enableEfs", and it is already inside. See line #499 "<#if enableEfs>", and line #562 "</#if>". If it is not inside, the integration tests won't pass.